components/bear-market-airdrop/Blocks/Hero/index.tsx

```diff
@@ -0,0 +1,400 @@
+/* eslint-disable no-restricted-imports */
+
+/* eslint-disable no-console */
+
+/* eslint-disable line-comment-position */
+
+/* eslint-disable react/jsx-key */
+import ClaimAirdrop from "./ClaimAirdrop";
+import Email from "./Email";
+import OpenPack from "./OpenPack";
+import Supply from "./Supply";
+import Unboxed from "./Unboxed";
+import {
+  Box,
+  Button,
+  Flex,
+  Heading,
+  Image,
+  Spinner,
+  Text,
+  useToast,
+} from "@chakra-ui/react";
+import { Chain, Fantom, Polygon } from "@thirdweb-dev/chains";
+import {
+  ConnectWallet,
+  MediaRenderer,
+  useAddress,
+  useContract,
+  useNetworkMismatch,
+  useOwnedNFTs,
+  useSDK,
+  useSwitchChain,
+} from "@thirdweb-dev/react";
+import {
+  SnapshotEntryWithProof,
+  TransactionResult,
+  fetchSnapshotEntryForAddress,
+} from "@thirdweb-dev/sdk";
+import axios from "axios";
```

Comment:
let's just use fetch instead of installing axios for this

Reply:
oh it seems it's not being used?

Reply:
ill remove it